bode/bode/resources/tasks/schemas.py

```diff
@@ -19,3 +19,10 @@ class TaskSchema(BaseSchema):
     description = fields.String()
     due_date = fields.DateTime()
     is_done = fields.Boolean()
+
+
+class TasksRelationSchema(BaseSchema):
+    id = fields.UUID(dump_only=True)
+    first_task_id = fields.String()
+    second_task_id = fields.String()
+    type = fields.String()
```

Comment:
This class is duplicate of `SimpleTaskRelationSchema` in [task_relations/schemas.py](bode/bode/resources/task_relations/schemas.py). I think It's better to use class `SimpleTaskRelationSchema`, because this schema is in the same directory as file [api.py](bode/bode/resources/task_relations/api.py).